index.js

```diff
@@ -20,28 +89,64 @@
  * @returns boolean, valid or invalid
  */
 export function validate(str, vars) {
+  // Operator Validation
   var len = str.length;
-  let funs = [
-    "CEILING",
-    "FLOOR",
-    "ABS",
-    "TRUNCATE",
-    "EXP",
-    "SQRT",
-    "LN",
-    "LOG",
-    "ceiling",
-    "floor",
-    "abs",
-    "truncate",
-    "exp",
-    "sqrt",
-    "ln",
-    "log"
-  ];
+  var op = "";
+  // Loop to identify operator strings
+  for (var j = 0; j < len; j++) {
```

Comment:
Not a big deal but by convention j is used for second level of nesting in loops. Although here using var the declaration would be hoisted. (Don't have to change it)